plugins/inline-row-separators/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@blockly/inline-row-separators",
+  "version": "1.0.0-beta.0",
```

Comment:
Does this need a beta?

Reply:
Rachel had originally requested this plugin be marked as beta, serving as a proof-of-concept for eventually adding a similar feature to core Blockly. I think that's still the plan? But other stuff about the design has changed, not sure if that makes a difference to how it should be tagged here.

Reply:
Update: she says to call it: 0.1.0